templates/encounter/forms/navbar.html.twig

```diff
@@ -0,0 +1,63 @@
+{% block navbar %}
+<nav class="navbar navbar-expand-lg navbar-light bg-light">
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+    </button>
+
+    <span class="navbar-brand">
+        {{ encounterDate|text }} {{ (groupEncounter == true) ? "Group Encounter"|xlt : "Encounter"|xlt}} {{ "for"|xlt }} {{ patientName|text }}
+    </span>
+
+    <div class="collapse navbar-collapse" id="navbarSupportedContent">
+        <ul class="navbar-nav mx-auto">
+            {% for category, details in menuArray %}
+                {% if details.children is iterable %}
+                    <li class="nav-item dropdown">
+                        <a class="nav-link dropdown-toggle" href="#" id="category_{{ category|xla }}" role="button" data-toggle="dropdown" aria-expanded="false">
```

Comment:
guessing you meant to use attr instead of xla (also noted this in several other places below where category is shown in a attribute where prob meant to use attr instead of xla)